### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by dmc-au**
Got a container up and running so that dependency management should be smooth.

Instructions are in README_Docker.md

### CODE REVIEW

**Comment by dmc-au** [comment]
Make sure you've got the docker daemon running before executing the commands in the README_Docker.md file

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-buster
+
+# Apt-get update and initial installations
+RUN apt-get update && apt-get install 
+RUN apt-get install vim -y --no-install-recommends
```
Comment (michaellisitsa) [inline Dockerfile:5]: Don't think you need Vim editor?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# RUN apt-get install nodejs npm -y 
+
+# Copy source files and install py packages
+RUN mkdir -p /opt/flydar
+RUN mkdir -p /opt/flydar/pip_cache
+COPY . /opt/flydar
```
Comment (michaellisitsa) [inline Dockerfile:11]: This adds the current sqlite file in your code repo into the docker container, together with any existing records. Instead, we should probably use a docker volume (via a docker-compose.yml). In the past I've created a postgres DB, and done the setup as per here https://github.com/michaellisitsa/expense_tracker/blob/0354eed5463653777f5731008043bd3152128f55/project/settings.py#L134-L144
and added the environment variables in the `docker-compose.yml` file:
https://github.com/michaellisitsa/expense_tracker/blob/0354eed5463653777f5731008043bd3152128f55/docker-compose.yml#L8-L11

It looks like I've probably followed the official docker django docs to get the above here https://docs.docker.com/samples/django/

**Review by michaellisitsa — CHANGES_REQUESTED** [review]
good job getting this worked out. The database stuff is probably the key thing, if you don't have a sqlite file in your directory beforehand and all migrations been run, I don't think the app will work. And the whole point of docker is that you can just clone the repo, and it works. i.e. you don't need to install python locally as well to do the db creation / migration.

**Pull request merged**